api.swagger.yaml

```diff
@@ -2679,6 +2679,10 @@ components:
               ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
             example: 01234567-89ab-cdef-1234-56789abcdef0
             type: string
+          description:
+            description: The HTML-rendered main body of the post.
```

Comment:
Maybe we should add something along the lines of 'Do not use this field for editing content'.